### DISCUSSION TITLE
Error manejo de inputs con SAVE

### DISCUSSION THREAD

**Discussion by Consolo0** [IIC2133-PUC/2024-1#90] [Tarea 1] [answered]
Hola, en la parte 2 cuando manejo los inputs, la cuál es un código que ya viene escrito, usando el archivo input de los testcases me pone que hay un error leyendo los datos de la cabeza 0, cual puede ser el error

**Comment by sebacarrascop** [comment, marked as answer]
hola!, revisa la issue #72
